packages/bruno/PKGBUILD

```diff
@@ -0,0 +1,100 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+# Initial PKGBUILD from AUR.
+# Old Maintainer: Vianney Bouchaud <aur dot vianney at bouchaud dot org>
+
+pkgname=bruno
+pkgver=1.34.2
+pkgrel=2
+pkgdesc='Opensource API Client for Exploring and Testing APIs.'
+groups=('blackarch' 'blackarch-webapp')
+arch=('x86_64')
+url="https://www.usebruno.com/"
+license=('MIT')
+_electron=electron31
+depends=(
+    "${_electron}"
+    'alsa-lib'
+)
+makedepends=(
+    'nvm'
+    'asar'
+    'python-setuptools'
+    'python'
+)
+
+source=(
+   "${pkgname}-${pkgver}.tar.gz::https://github.com/usebruno/bruno/archive/v${pkgver}.tar.gz"
+   "com.usebruno.app.Bruno.desktop"
+)
+
+sha512sums=('de75210f37907bea641b0f93cd1511b3024e27bbacd892a3de8918b514ca1e1e012fbb553a37095b72f397b3ca87b1f7d5be62c7d7b68616e3b0178dade8facf'
+            '0c303dd2a3cb1d1628d25b33d515df4fdaf116298520bed5a251d8320d5f771146a625165ab07fbbf7f4698ebc3bcfc7682a44aaa635d980493c7fc208389884')
+
+_ensure_local_nvm() {
+  # lets be sure we are starting clean
+  which nvm >/dev/null 2>&1 && nvm deactivate && nvm unload
+
+  export NVM_DIR="${srcdir}/${pkgname}-${pkgver}/.nvm"
```

Comment:
IS nvm really needed to build an electron app?

https://wiki.archlinux.org/title/Electron_package_guidelines

Reply:
Not really needed. I removed it. I can leave nvm package to be added in BA, if it could be useful for the future for other stuff.